src/Uno.UI/WasmCSS/Uno.UI.css

```diff
@@ -26,7 +26,7 @@ body {
     margin & padding are calculated through Xaml elements.
   */
   margin: 0 !important;
-  padding: 0 !important;
+  padding: 0;
```

Comment:
Si enlever le tag !important fait que le padding fonctionne, est-ce que ca vx dire que les marges, le line-height et les autres proprité css ici ne fonctionne pas aussi? Peut-être que c'est l'effet escompté aussi je demande par curiosité,

Reply:
Bien vu @alextrepa, je me posais la meme question aussi ;)

Reply:
Curious +1, I was under the impression that Xaml Margin, Padding etc were handled entirely by Uno and the absolute position of the element was being set directly.

Reply:
It was there because the HTML Padding were not supposed to be used at all in the first versions of Uno-WASM, since it's the responsibility of the layour engine.

But in the case of TextBlock, it's a legitimate reason.